src/lua/uri.lua

```diff
@@ -60,6 +60,41 @@ local uri_set_stash = buffer.ffi_stash_new('struct uri_set')
 local uri_set_stash_take = uri_set_stash.take
 local uri_set_stash_put = uri_set_stash.put
 
+-- RFC 3986: Uniform Resource Identifier (URI): Generic Syntax
+-- 2.2. Reserved Characters
+-- https://datatracker.ietf.org/doc/html/rfc3986.html#section-2.2
+local UNRESERVED_RFC3986 = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ-._~"
+
+local RFC3986 = {
+    unreserved = UNRESERVED_RFC3986,
```

Comment:
I'm completely for using existing terminology, when it exists. Especially, when it is introduced by a standard. But this case is special for me: too weird term. I need to remember myself again and again that `unreserved` is `unsafe`. Can we call it just `unsafe`? At least it'll make the code itself much more clear. But also it'll make the API more clear. Nothing prevent us from saying 'unreserved character set from RFC 3986 is picked up for `unsafe` by default' in the documentation to make pedants happy.

Reply:
Sure. It is what I said about.